### DISCUSSION TITLE
Problem uploading image

### DISCUSSION THREAD

**Discussion by Kulturserver** [OpnForm/OpnForm#592] [Q&A]
Unfortunately we still have another problem - when uploading an image, the path doesn't seem to be set correctly. 
We bought a domain & configured it according to configure/custom_domain in the api/.env like this:

APP_URL=https://forms.culturebase.org/
client/.env:
NUXT_PUBLIC_APP_URL=https://forms.culturebase.org/
NUXT_PUBLIC_API_BASE=https://forms.culturebase.org/api

If we upload an image now, it is displayed as corrupt. The link is like this:
http://127.0.0.1:8081/forms/assets/image_name.jpg

When changing the IP and port to forms.culturebase.org the image is displayed correctly. What did we miss? Any help would be appreciated, thanks!

**Comment by JhumanJ** [comment]
Hey, have you restarted the docker setup after changing the .env variables?

**Reply by Kulturserver** [reply]
Yes, we did!

**Reply by JhumanJ** [reply]
Do you have the latest docker version?

**Reply by Kulturserver** [reply]
Yes, we do.

**Reply by JhumanJ** [reply]
Random thought, can you try removing the trailing slash in your `.env` files?

**Reply by Kulturserver** [reply]
We did so now, but unfortunately, it didn't help.

**Reply by Kulturserver** [reply]
Unfortunately, we still encounter the problem, even with the latest updates. One additional question is: How can we definitely see which version of opnform is installed? I can find no notice in the GUI and the docker files are all just called "latest".

**Comment by Kulturserver** [comment]
Here is a more specific recap of our problem, we notice the difference between the setting in the client/.env whether a real url is set or not.

1. first environment without domain http://servername.org:8080/forms/form-name:
 - http://servername.org:8080/forms/form-name/show/share works
 - Preview shows not working url : http://localhost/forms/form-name
 - File upload works with correct path: http://servername.org:8080/forms/assets/logo_4.png or http://servername.org:8080/open/forms/5/submissions/file/04.jpeg
 
2. second environment with domain https://forms.culturebase.org/forms/form-name
 - https://forms.culturebase.org/forms/form-name/show/share works
 - Preview shows working url: https://forms.culturebase.org/forms/form-name
 - File upload works, but wrong path is set: http://127.0.0.1:8081/forms/assets/logo_4.png or http://127.0.0.1:8081/open/forms/5/submissions/file/04.jpeg
 - with correct path, images can be displayed http://forms.culturebase.org/open/forms/5/submissions/file/04.jpeg

Here are our configurations:
[api.env.txt](https://github.com/user-attachments/files/18455565/api.env.txt)
[client.env.txt](https://github.com/user-attachments/files/18455566/client.env.txt)

**Reply by Kulturserver** [reply]
I'm afraid we also still looking for a solution in this matter - maybe there is also a connection to #671 ? Any hints will be appreciated, thanks!

**Comment by JhumanJ** [comment]
Hey guys - can you please try with the latest docker image?

**Comment by Kulturserver** [comment]
We tried - but as mentioned in #671 we run into difficulties installing it also.

**Comment by Kulturserver** [comment]
Unfortunately, the problems remain with the new docker image, see #671

**Comment by Kulturserver** [comment]
Any help or ideas for this issue would be highly appreciated!

**Comment by JhumanJ** [comment]
Apologies for the late replied - this should not be fixed!

Just updated the docker images. Make sure to get the latest version of the docker-compose file when updating 🙂
Also worse checking the .env.docker of both api and client for available options.
